crucible-mir/src/Mir/PP.hs

```diff
@@ -77,7 +77,7 @@ instance Pretty Ty where
     pretty (TyAdt _ origDefId tys)    = pr_id origDefId <> pretty tys
     pretty (TyFnDef defId)       = pretty "fnDef" <+> pr_id defId
     pretty (TyClosure tys)       = pretty "closure" <+> pretty tys
-    pretty TyCoroutine = pretty "coroutine"
+    pretty (TyCoroutine ca) = pretty "coroutine" <+> viaShow ca
```

Comment:
I do think it would be worth adding a `Pretty CoroutineArgs` instance and using it here (and in the code for pretty-printing `AKCoroutine`s below) rather than relying on the `Show` instance, which will likely look pretty jarring when intermixed with other pretty-printed MIR code.